spec/docs/termmap.md

```diff
@@ -21,15 +21,15 @@ if it is a rule that specifies how the [=RDF triple=]'s object is generated; and
 * a [=graph map=] (`rml:GraphMap`),
 if it is a rule that specifies how the [=RDF triple=]'s [=named graph=] is generated.
 
-A [=term map=] generates different RDF terms depending on the position of the [=term map=] in the [=RDF triple=]:
+A [=term map=] generates different types of [=RDF terms=] depending on the position of the [=term map=] in the [=RDF triple=]:
 * a [=subject map=] (`rml:SubjectMap`)
 is a rule that MUST generate either an [=IRI=] or a [=blank node=];
 * a [=predicate map=] (`rml:PredicateMap`)
 is a rule that MUST generate an [=IRI=];
 * an [=object map=] (`rml:ObjectMap`)
 is a rule that MUST generate an [=IRI=], a [=blank node=] or a [=literal=];
 * a [=graph map=] (`rml:GraphMap`)
```

Comment:
graph maps are not 'defined', the reference doesn't work

Reply:
Indeed. The graph maps section still needs to be written.